docs/api/trinity/index.rst

```diff
@@ -0,0 +1,53 @@
+Trinity
+=======
+
+This section aims to provide a detailed description of all APIs. If you are looking for something more hands-on or higher-level check out the existing `Trinity guides </guides/trinity>`_..
```

Comment:
Probably worth adding a `.. warning::` to this page indicating that the trinity API is actively being developed and likely to change regularly.

Reply:
We can be pretty aggressive here to drive the point home: "We expect each alpha release to have breaking changes to the API."